frontend/src/streamlist/StreamsTable.jsx

```diff
@@ -1,62 +1,129 @@
 import React from "react";
-import {
-  Box,
-  Table,
-  TableBody,
-  TableContainer,
-  TableHead,
-  TableRow
-} from "@material-ui/core";
+import { Box, TableContainer, Typography } from "@material-ui/core";
 import PropTypes from "prop-types";
-import HeadCells from "./HeadCells";
-import SingleStreamRow from "./SingleStreamRow";
 
-export default class StreamsTable extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      streams: []
-    };
-    this.dataSource = props.dataSource;
-    this.handleStreamsChange = this.handleStreamsChange.bind(this);
-  }
+import {
+  ExpandLess,
+  ExpandMore,
+  ArrowDownward,
+  FirstPage,
+  LastPage,
+  ChevronRight,
+  ChevronLeft
+} from "@material-ui/icons";
 
-  componentDidMount() {
-    this.dataSource.getAllStreams(this.handleStreamsChange);
-  }
+import MaterialTable from "material-table";
 
-  handleStreamsChange(streams) {
-    this.setState({
-      streams
-    });
-  }
+import StatusIndicator from "../general/StatusIndicator";
+import ActionButtons from "./ActionButtons";
+import StreamInfo from "../model/StreamInfo";
+
+function getColumnInfo() {
+  return [
+    {
+      title: "ID",
+      field: "id"
+    },
+    {
+      title: "Date",
+      field: "date"
+    },
+    {
+      title: "Sender",
+      field: "sender"
```

Comment:
was this not working with `sender.name`? & thus the change to the `StreamInfo` model?

Reply:
Actually I could just make the fields in the column info to just be `sender.name` and `receiver.name`, so I'll modify that.